### PR TITLE
AR visualization

### PR DESCRIPTION
**Pull request by pekkaran**
Also @kaatrasa

This PR adds example mesh and point cloud visualizations for the SDK Mapping API. It works live on OAK-D and with recorded sessions.

The example is invoked through `mapping_ar.py` which internally uses two separate classes in the `mapping_ar_renderers/` directory:
```
├── mapping_ar.py
└── mapping_ar_renderers
    ├── mesh.py
    ├── mesh.frag
    ├── mesh.vert
    ├── point_cloud.py
    ├── point_cloud.frag
    └── point_cloud.vert
```

The commits are probably best viewed together.

Requires SDK features not released yet, perhaps should not be merged before the next release.

### CODE REVIEW

File: python/oak/mapping_visu.py [hunk 1]
```diff
@@ -173,21 +173,33 @@ def removeKeyFrame(self, keyFrameId):
...
+    configInternal = {
+        "computeStereoPointCloud": "true",
+        "pointCloudNormalsEnabled": "true",
+        "computeDenseStereoDepth": "true",
+    }
+    if args.dataFolder and args.useRectification:
+        configInternal["useRectification"] = "true"
+    else:
+        configInternal["alreadyRectified"] = "true"
+
```
Comment (pekkaran) [inline python/oak/mapping_visu.py:202]: This unrelated change makes the `mapping_visu.py` example work more reliably with recordings from different devices.

**Review by oseiskar — APPROVED** [review]
